mobile/src/components/uam-configure-group/components/permissions-table/permissions-table.tsx

```diff
@@ -9,7 +9,7 @@ import {
 } from '~/components/uam-configure-group/helpers/helpers';
 import { useAppForm } from '~/hooks/hooks';
 
-import { styles } from './styles';
+import { styles } from '../styles';
```

Comment:
Styles should be near the component

Reply:
styles are used for two components

Reply:
> Styles should be near the component

styles are used for two components

Reply:
Then we will have two identical files with styles.
Is it ok?

Reply:
According to this comment, we can have duplicate styles
https://github.com/BinaryStudioAcademy/bsa-2022-guruhub/pull/34#discussion_r940171938

Reply:
I think Oleksandr Maiboroda's version is better

Reply:
ok i will change it

Reply:
I agree with @fshabanov, it's ok to duplicate styles here. If we don't want to duplicate them, we need to create a common component, but we don't need this in our case